### DISCUSSION TITLE
Specific workflow runs keeps failing with an internal error on macOS

### DISCUSSION THREAD

**Discussion by FranciscoPombal** [community/community#25490] [Actions]
<p>In these 2 runs, some macOS jobs keep failing with an “internal error” no matter how many times I re-run them:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/qbittorrent/qBittorrent/actions/runs/277763652" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/17580742/181091373-4482b1d2-b413-4296-b81f-a37bc1af8fc3.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/qbittorrent/qBittorrent/actions/runs/277763652" target="_blank" rel="noopener nofollow ugc">qbittorrent/qBittorrent</a></h3>

<p>qBittorrent BitTorrent client. Contribute to qbittorrent/qBittorrent development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<br>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/qbittorrent/qBittorrent/actions/runs/277637794" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/17580742/181091374-674b7eed-fb50-43d6-b1de-aad92740fe6c.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/qbittorrent/qBittorrent/actions/runs/277637794" target="_blank" rel="noopener nofollow ugc">qbittorrent/qBittorrent</a></h3>

<p>qBittorrent BitTorrent client. Contribute to qbittorrent/qBittorrent development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I have tried a few days/weeks ago, as well as right now, same result.</p>
<p>The jobs don’t even start, they just go straight to showing “This check has failed” with a big “X” in the middle of the job log. What’s going on?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/franciscopombal">@FranciscoPombal</a>,</p>
<p>According to the matrix configurations you set for “<strong>ci_macos</strong>”,</p>
<pre><code class="lang-yaml">jobs:
  . . .

  ci_macos:
    name: macOS + vcpkg
    runs-on: macos-10.15
    strategy:
      matrix:
        qbt_gui: ["GUI=ON", "GUI=OFF"]
      fail-fast: false
</code></pre>
<p>it should only generate 2 matrix jobs:</p>
<ul>
<li>
<strong>macOS + vcpkg (GUI=ON)</strong> runs on <strong>macos-10.15</strong>
</li>
<li>
<strong>macOS + vcpkg (GUI=OFF)</strong> runs on <strong>macos-10.15</strong>
</li>
</ul>
<p>Looks like, the failed jobs are duplicated and extra.</p>
<p>I have created an issue ticket (<a href="https://github.com/actions/virtual-environments/issues/1738" rel="noopener nofollow ugc">actions/virtual-environments#1738</a>) to help you report the problem to the appropriate engineering team for further investigation and evaluation.<br>
you can follow this issue ticket and add your comments to it.</p>

**Comment by FranciscoPombal** [comment]
<p><a class="mention" href="/brightran">@brightran</a> I hadn’t noticed that the jobs were duplicated! Bad case of tunnel vision I guess. It is strange why this only happens in these two instances, and not on any other run. I hope the Actions team can find and fix the bug, whatever it is. Thanks for creating the issue, I’ll post there from now on.</p>
